<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widgetLockscreen" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>377</width>
      <height>196</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hLayoutLockScreen">
     <item>
      <layout class="QVBoxLayout" name="vLayoutLeft">
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="lblAvatar">
         <property name="text">
          <string>Avatar</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vLayoutRight">
       <item>
        <widget class="QLabel" name="lblUsername">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblLogged">
         <property name="text">
          <string>Has Logged In</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblPrompt">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hLayoutPWD">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEditPassword"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnUnlock">
           <property name="text">
            <string>Unlock</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
